lib/fork.js

```diff
@@ -50,14 +50,14 @@ module.exports = function (file, opts) {
 	var relFile = path.relative('.', file);
 
 	var exiting = false;
-	var send = function (ps, name, data) {
```

Comment:
Really, we could do away with `wrap-send` entirely, and just inline in `send-to-parent` and `fork.js`. It's hardly worth it at this point.